tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -2422,48 +2477,57 @@ class RelatedIdScanner : public SourceEntityWalker {
 } // end anonymous namespace
 
 void SwiftLangSupport::findRelatedIdentifiersInFile(
-    StringRef InputFile, unsigned Offset, bool CancelOnSubsequentRequest,
-    ArrayRef<const char *> Args, SourceKitCancellationToken CancellationToken,
+    StringRef PrimaryFile, StringRef InputFile, unsigned Offset,
```

Comment:
`Name` is kind of weird to me since for a path on disk it implies the name rather than full path. I'd somewhat expect the contents itself to be called `<whatever>FileContents`. How strongly do you feel about this?

Reply:
`Path` has similar issues, ie. it implies a path on disk when `InputFile` can actually just be the buffer name. These could be `PrimaryFilePath` and `InputBufferName` though, since the former is always a path and the latter is meant to be a buffer name/identifier, not a path/filename. Time for lots of renaming 😅

Reply:
I renamed them all to `PrimaryFilePath` and `InputBufferName`. Hopefully that's more clear.